tests/Integration/Behaviour/Features/Scenario/Product/update_stock_classic.feature

```diff
@@ -15,10 +15,10 @@ Feature: Update product stock from Back Office (BO)
     And shop configuration for "PS_ADVANCED_STOCK_MANAGEMENT" is set to 0
 
   Scenario: I check default stock values
-    Given I add product "product1" with following information:
+    When I add product "product1" with following information:
```

Comment:
Not related to this line But there are still some behat test without stock movement which are likely to fail now since the movement is always applied (L155)